lua/mason-registry/glint-language-server/init.lua

```diff
@@ -0,0 +1,16 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "glint-language-server",
+    desc = [[Glint Language server, for Glimmer-flavored JavaScript and TypeScript]],
+    homepage = "https://github.com/typed-ember/glint/tree/main/packages/core",
+    categories = { Pkg.Cat.LSP },
+    languages = {
+        Pkg.Lang.Handlebars,
+        Pkg.Lang.Glimmer,
+        Pkg.Lang["typescript.glimmer"],
+        Pkg.Lang["javascript.glimmer"],
```

Comment:
These language entries are meant for discovery purposes only, so these compound-like filetype names are not used elsewhere. I wonder if it'd be enough with "Handlebars" and "Glimmer"?

Reply:
ah, ok, probably -- I thought they needed to somehow map to a file's `filetype`

Reply:
The "programmatic" filetype mapping is governed by [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/server_configurations/glint.lua#L24-L31) for LSP servers! These language entries are really just arbitrary strings only used for presentational purposes, for now.

Reply:
My latest commit (on both repos) brought back 
```
[mason-lspconfig.nvim] Server "glint" is not a valid entry in ensure_installed. Make sure to only provide lspconfig server names.
```
:thinking:

Reply:
ok, latest _latest_ commit re-fixed this.